test/extended/networking/network_segmentation.go

```diff
@@ -451,6 +585,91 @@ func waitForUserDefinedNetworkReady(namespace, name string, timeout time.Duratio
 	return err
 }
 
+func newPrimaryUserDefinedNetworkManifest(name string) string {
+	return `
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: ` + name + `
+spec:
+  topology: Layer3
+  layer3:
+    role: Primary
+    subnets:
+    - cidr: 10.20.100.0/16
+    - cidr: 2014:100:200::0/60
+`
+}
+
+func newUserDefinedNetworkManifest(name string) string {
+	return `
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: ` + name + `
+spec:
+  topology: "Layer2"
+  layer2:
+    role: Secondary
```

Comment:
Since there tests are about primary network, should we change this manifest to specify role=Primary?